exporter/elasticsearchexporter/exporter.go

```diff
@@ -132,11 +133,69 @@ func (e *elasticsearchExporter) pushLogRecord(ctx context.Context, resource pcom
 
 	document, err := e.model.encodeLog(resource, record, scope)
 	if err != nil {
-		return fmt.Errorf("Failed to encode log event: %w", err)
+		return fmt.Errorf("failed to encode log event: %w", err)
 	}
 	return pushDocuments(ctx, fIndex, document, e.bulkIndexer)
 }
 
+func (e *elasticsearchExporter) pushMetricsData(
+	ctx context.Context,
+	metrics pmetric.Metrics,
+) error {
+	var errs []error
+
+	rls := metrics.ResourceMetrics()
+	for i := 0; i < rls.Len(); i++ {
+		rl := rls.At(i)
+		resource := rl.Resource()
+		ills := rl.ScopeMetrics()
```

Comment:
nit: not clear what "ills" is (is this copied from logs? instruction scope logs?), could we give it a more meaningful name such as "scopeMetrics"?

Reply:
Good point, I actually hate it when variable names don't make it easy to understand the code. Guilty as charged!
Fixed in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33513/commits/1136e74c2d4a675f42c24b2b65c09873367711d4.